provider/execution/execution.go

```diff
@@ -225,6 +226,9 @@ func (e TableExecutor) cleanupStaleData(ctx context.Context, client schema.Clien
 
 // callTableResolve does the actual resolving of the table calling the root table's resolver and for each returned resource resolves its columns and relations.
 func (e TableExecutor) callTableResolve(ctx context.Context, client schema.ClientMeta, parent *schema.Resource) (uint64, diag.Diagnostics) {
+	clock := stats.DefaultEngine.Clock("callTableResolve", stats.Tag{Name: "table", Value: e.Table.Name})
```

Comment:
We can add more tags if we'd like.
`clock.Stop` add another measurement with the duration, so we can track what operations completed and how long they took